sympy/matrices/common.py

```diff
@@ -2019,6 +2019,71 @@ def replace(self, F, G, map=False, simultaneous=True, exact=None):
         return self.applyfunc(
             lambda x: x.replace(F, G, map=map, simultaneous=simultaneous, exact=exact))
 
+    def rot90(self, k=1):
+        """Rotates Matrix by 90 degrees
+
+        Parameters
+        ==========
+
+        k : int
+            Parameter k specifies the rotation
+            k is 0 when the Matrix is not rotated
+            k is +ve when the Matrix is rotated Clockwise
+            k is -ve when the Matrix is rotated Anti-Clockwise
+            |k| > 1 represent iterative rotations
+            i.e.
+            2 --> Clockwise x 2
+            -2 --> Anti-Clockwise x 2
+
+        Examples
+        ========
+
+        >>> from sympy import MatrixSymbol
+        >>> A = MatrixSymbol('A', 2, 2).as_explicit()
```

Comment:
Would simple letters be less busy? 
```python
>>> Matrix(2,2,symbols('A:2:2'))
Matrix([
[A00, A01],
[A10, A11]])
>>> Matrix(2,2,symbols('a:d'))
Matrix([
[a, b],
[c, d]])
```

Reply:
I would just show k=1 and k=-2.

Reply:
alrighty 😄 ... thanks ... will add the changes

Reply:
> Would simple letters be less busy?
> 
> ```python
> >>> Matrix(2,2,symbols('A:2:2'))
> Matrix([
> [A00, A01],
> [A10, A11]])
> >>> Matrix(2,2,symbols('a:d'))
> Matrix([
> [a, b],
> [c, d]])
> ```

I'd go with a,b,c,d. They seem simple. wdys?